hasher-matcher-actioner/hmalib/lambdas/api/status_api.py

```diff
@@ -2,30 +2,76 @@
 
 import logging
 import os
-
+import bottle
 import boto3
 import json
+from boto3.dynamodb.conditions import Attr
+from apig_wsgi import make_lambda_handler
 
+app = bottle.default_app()
+apig_wsgi_handler = make_lambda_handler(app)
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
-# s3_client = boto3.client("s3")
-# dynamodb  = boto3.resource("dynamodb")
+s3_client = boto3.client("s3")
+dynamodb = boto3.resource("dynamodb")
+
+DYNAMODB_TABLE = os.environ["DYNAMODB_TABLE"]
+
+
+@app.get("/echo/<val>")
+def echo(val):
+    return val
+
+
+@app.get("/")
+def root():
+    return {
+        "reply": "Hello World, HMA",
+    }
+
+
+@app.get("/matches")
+def matches():
+    results = check_db_for_matches()
+    logger.info(results)
+    matches = []
+    for match in results:
+        matches.append({match["PK"]: match["SK"]})
+    return json.dumps({"matches": matches})
 
-# DYNAMODB_TABLE = os.environ["DYNAMODB_TABLE"]
+
+@app.get("/hash/<key>")
+def hashes(key=None):
+    results = check_db_for_hash(key)
+    logger.info(results)
+    return json.dumps(results)
 
 
 def lambda_handler(event, context):
     """
-    Status of delpoyed HMA 
+    Status of delpoyed HMA
```

Comment:
When this was a "hello world" the term of art "status" was being used. I'm wondering if at this point this file should be called api.py? Eventually when we break things up into .py's that map to different categories of endpoints we'll have different names that tie to the category.